currency_rate_update_boi/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+This module adds `Banca d'Italia <https://tassidicambio.bancaditalia.it/>`_
+currency exchange rates provider.
+
+Data export instructions are available at
+https://tassidicambio.bancaditalia.it/terzevalute-wf-ui-web/assets/files/Operating_Instructions.pdf
+
+Exchange rates are exported in JSON format.
```

Comment:
Se ti va, visto che il modulo è nel repository l10n-italy, sarebbe ottimo se il readme avesse anche la sezione in italiano.

Reply:
Fatto!